FastSimulation/Tracking/plugins/TrajectorySeedProducer.cc

```diff
@@ -164,69 +159,90 @@ TrajectorySeedProducer::beginRun(edm::Run const&, const edm::EventSetup & es)
     magneticFieldMap = &(*magneticFieldMapHandle);
     trackerTopology = &(*trackerTopologyHandle);
 
-    thePropagator = new PropagatorWithMaterial(alongMomentum,0.105,magneticField); 
+    thePropagator = std::move(std::shared_ptr<PropagatorWithMaterial>(new PropagatorWithMaterial(alongMomentum,0.105,magneticField)));
```

Comment:
@matt-komm 

I don't really get the consequence of this change.
Does it may be ensure that the previous thePropagator is deleted at every new beginRun?